apps/condo/access.js

```diff
@@ -31,56 +31,36 @@ const rules = {
       Organization
 
     */
-    canReadOrganizations: isSignedIn,
-    canManageOrganizations: ({ authentication: { item: user } }) => {
-        if (!user) return false
-        if (user.isAdmin) return true
-        return {
-            // user is inside employee list
-            employees_some: { user: { id: user.id, canManageOrganization: true } },
-        }
-    },
-    canReadEmployees: ({ authentication: { item: user } }) => {
+    canReadRoles: ({ authentication: { item: user } }) => {
         if (!user) return false
         if (user.isAdmin) return {}
-        return {
```

Comment:
Why you delete read roles access check?

Reply:
`canReadRoles` is domain specific, I though it's place in organization domain.